types/react-datepicker/index.d.ts

```diff
@@ -15,7 +15,7 @@
 //                 Kerry Gougeon <https://github.com/kerry-g>
 //                 Walter Kennedy <https://github.com/wthefourth>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
-// TypeScript Version: 2.8
+// TypeScript Version: 3.1
```

Comment:
Why's this need 3.1? You can probably just remove this now and let it implicitly rely on 2.8

Reply:
It needed 3.1 for the `(...args: any)`, but I'm taking that part out most likely. I'll try and get a new PR submitted ASAP.